### DISCUSSION TITLE
Option to choose the font

### DISCUSSION THREAD

**Discussion by jbonadiman** [go-shiori/shiori#612] [Feature requests]
I wish there was an option to choose the font when reading the article on Shiori.
I know it's possible to achieve this using an extension like Stylus, but I think it would be nice to have the possibility to have that as a setting for each user.

**Comment by fmartingr** [comment]
Would the idea here having a set of predefined fonts to choose from? Choosing custom fonts?

Could something like https://github.com/go-shiori/shiori/discussions/338 work for you as well?

**Comment by jbonadiman** [comment]
It would be nice to have predefined fonts and the option to bring your own.
And yes, #338 would work for me.
